site/docs/red-team/strategies/index.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_label: Overview
+---
+
+# Red Team Strategies
+
+## What are Strategies?
+
+Strategies are specialized modifications made on top of plugin test cases to test different attack vectors. They generally increase the Attack Success Rate (ASR) of the intents generated by plugins. The are applied during redteam generation (before eval). Strategies can be applied to individual plugins or to the entire test suite.
+
+## How to Select Strategies
+
+Before selecting strategies you should understand if you are building a single-turn or multi-turn application. Single-turn and multi-turn LLM applications differ primarily in their handling of context.
+
+### Single-turn Applications
```

Comment:
I think these paragraphs are too dense - I'd at a minimum break them up into separate paragraphs, add images, etc for readability